packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/clientcreds/use-flow.md

```diff
@@ -6,7 +6,7 @@ If you are not using existing libraries, you can make a direct request to Okta's
 
 The Client Credentials flow is intended for server-side (confidential) client applications with no end user, which normally describes machine-to-machine communication. Your client application needs to have its client ID and secret stored in a secure manner. You can find the client ID and secret on the **General** tab for your app integration.
 
-[Base64 encode](#base64-encode-the-client-ID-and-client-secret) the client ID and secret and then pass through [Basic Authentication](https://tools.ietf.org/html/rfc7617) in the request to your [Custom Authorization Server's](/docs/concepts/auth-servers/#custom-authorization-server) `/token` endpoint:
+Base64 encode the client ID and secret (as shown below) and then pass through [Basic Authentication](https://tools.ietf.org/html/rfc7617) in the request to your [Custom Authorization Server's](/docs/concepts/auth-servers/#custom-authorization-server) `/token` endpoint:
```

Comment:
```suggestion
[Base64 encode the client ID and secret](#base64-encode-the client-id-and-client-secret) and then pass through [Basic Authentication](https://tools.ietf.org/html/rfc7617) in the request to your [Custom Authorization Server's](/docs/concepts/auth-servers/#custom-authorization-server) `/token` endpoint:
```
sorry for this nit-pick: our style guide tells us to avoid using "below" and use a link, or use "later" or "the following"

Reply:
I used this because the link wouldn't work (there seems to be a weird bug with linking to it). I've updated it to "later" to be more correct as per style guide.